README.md

```diff
@@ -191,59 +191,64 @@ In that case, the handler body is taken from a file in the
 
 ## Route handlers
 
-A route handler is a function which accepts one argument (**Request**) and
-returns one value (**Response**).
+A route handler is a function which accepts one argument (**Context**) and
+returns response object, which is set inside the **Context** object by the index of **res**.
 
 ```lua
-function my_handler(req)
-    -- req is a Request object
-    -- resp is a Response object
-    local resp = req:render({text = req.method..' '..req.path })
-    resp.headers['x-test-header'] = 'test';
-    resp.status = 201
-    return resp
+function my_handler(self)
+    -- self is a Context object
+    self:render({
+        status = 201,
+        headers = {
+            ['x-test-header'] = 'test'
+        },
+        text = self.req.method..' '..self.req.path
+    })
+
+    -- res object within Context will be handled automatically
 end
 ```
 
-### Fields and methods of the Request object
+### Fields and methods of the Context object
 
-* `req.method` - HTTP request type (`GET`, `POST` etc).
-* `req.path` - request path.
-* `req.query` - request arguments.
-* `req.proto` - HTTP version (for example, `{ 1, 1 }` is `HTTP/1.1`).
-* `req.headers` - normalized request headers. A normalized header
+* `ctx.req.body` - request's body
```

Comment:
This is a breaking change.

It will never be merged in `tarantool/http` v1.

Reply:
Okay. We just use our version. Current tarantool/http v1 not usefull for us. Notify me when v2 go to beta.